content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -93,17 +93,19 @@ for the kube-apiserver component.
 A discovery endpoint `/openapi/v3` is provided to see a list of all
 group/versions available. This endpoint only returns JSON. These group/versions
 are provided in the following format:
+
 ```json
```

Comment:
How I'd fix this: mark it as YAML, not JSON. The syntax highlighter for YAML is more forgiving, and every JSON document is valid YAML.

Also add a comment to explain why you're making JSON as YAML to the highlighter.